dockers/base-cuda/Dockerfile

```diff
@@ -74,26 +90,28 @@ ENV CONDA_DEFAULT_ENV=${CONDA_ENV}
 
 COPY ./requirements/extra.txt requirements-extra.txt
 COPY ./requirements/test.txt requirements-tests.txt
-COPY ./requirements/examples.txt requirements-examples.txt
 
 RUN \
     # Disable cache
     pip config set global.cache-dir false && \
     #echo ". ${WORKDIR}/miniconda/etc/profile.d/conda.sh" >> ~/.bashrc && \
     #echo "conda activate ${CONDA_ENV}" >> ~/.bashrc && \
     #source ~/.bashrc && \
+    # install NVIDIA AMP
```

Comment:
@ydcjeff :]

Reply:
Ouch it took 20mins like before

What abt skipping apex in py3.8 since we don't use py3.8 in Drone?

Reply:
we want to give Docker users full experience, that is the point to have installed all long-running tasks, also this mostly run in a nightly fashion/CI and at this moment Drone takes quite long with some queue...

Reply:
Okay 👌